stdeb.cfg

```diff
@@ -1,4 +1,5 @@
 [colcon-zsh]
 Depends3: python3-colcon-core (>= 0.3.0)
+Recommends3: zsh
```

Comment:
`colcon-common-extensions` currently depends on `colcon-zsh` (https://github.com/colcon/colcon-common-extensions/blob/a1ce25fbdb51431046bdac081f87cc366f5a19ab/setup.cfg#L47). With this recommends that would imply that installing `colcon-common-extensions` will also install `zsh`. I am not sure that is desired for all users.

An alternative if this recommends is really desired could be to remove `colcon-zsh` from the common extensions package?

Reply:
@Theosakamg Do you have any comment on this? Otherwise I might go ahead close this ticket (same for colcon/colcon-bash#13).